WPMedia/fusion-news-theme-blocks
# THEMES-1474: Input display bug

## Description

_Information about what you changed for this PR_

## Jira Ticket

- [THEMES-1474](https://arcpublishing.atlassian.net/browse/THEMES-1474)

## Acceptance Criteria

The search box should meet the margin size and cover the search icon.

## Test Steps

1. Checkout this branch `git checkout THEMES-1474`
2. Checkout `arc-themes-components` branch `git checkout THEMES-1474`
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/search-results-list-block,@wpmedia/identity-block`
4. Add the search results list block to a page, and make sure the input is sized appropriately.
5. Then check the Identity blocks that use the input component (Log in, sign up, account management) and make sure their inputs are still sized correctly according to the Figma docs.

## Effect Of Changes

### Before

![Screenshot 2023-11-02 at 1 31 34 PM](https://github.com/WPMedia/arc-themes-blocks/assets/4759882/16cb05cb-0a0f-4f41-9e93-7c5d4b822dbe)

### After

![Screenshot 2023-11-02 at 1 31 16 PM](https://github.com/WPMedia/arc-themes-blocks/assets/4759882/38c91aa2-2c6a-4754-bbbc-ba7fa43d33b7)

## Dependencies or Side Effects

- arc-themes-components PR: https://github.com/WPMedia/arc-themes-components/pull/217

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1474]: https://arcpublishing.atlassian.net/browse/THEMES-1474?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
